src/localconf.js

```diff
@@ -344,98 +344,105 @@ function buildConfig(opts) {
+              description: `This simulation runs a docker version of the Hypothesis Testing
           using Phylogenies (HyPhy) tool over data submitted in the NIAID Data Hub. \n
           The simulation is focused on modeling a Bayesian Graph Model (BGM) based on a binary matrix input.
           The implemented example predicts the virulence status of different influenza strains based on their mutations
           (the mutation panel is represented as the input binary matrix).`,
-            image: '/src/img/analysis-icons/virulence.png',
-          };
-          break;
-        case 'vaGWAS':
-          analysisApps.vaGWAS = {
-            title: 'vaGWAS',
-            description: 'Expression-based Genome-Wide Association Study',
-            image: '/src/img/analysis-icons/gwas.svg',
-            options: [
-              {
-                label: 'Lung',
-                value: 'Lung',
-              },
-              {
-                label: 'Gastrointestina',
-                value: 'Gastrointestina',
-              },
-              {
-                label: 'Prostate',
-                value: 'Prostate',
-              },
-              {
-                label: 'Head and Neck',
-                value: 'Head and Neck',
-              },
-              {
-                label: 'Skin',
-                value: 'Skin',
-              },
-              {
-                label: 'NULL',
-                value: 'NULL',
-              },
-              {
-                label: 'Lymph Node',
-                value: 'Lymph Node',
-              },
-              {
-                label: 'Liver',
-                value: 'Liver',
-              },
-              {
-                label: 'Musculoskeleta',
-                value: 'Musculoskeleta',
-              },
-              {
-                label: 'Occipital Mass',
-                value: 'Occipital Mass',
-              },
-              {
-                label: 'Brain',
-                value: 'Brain',
-              },
-              {
-                label: 'BxType',
-                value: 'BxType',
-              },
-            ],
-          };
-          break;
-        case 'GWASUIApp':
-          analysisApps.GWASUIApp = {
-            title: 'Gen3 GWAS',
-            description: 'Use this App to perform high throughput GWAS on Million Veteran Program (MVP) data, using the University of Washington Genesis pipeline',
-            image: '/src/img/analysis-icons/gwas.svg',
-          };
-          break;
-        case 'GWASResults':
-          analysisApps.GWASResults = {
-            title: 'GWAS Results',
-            description: 'Use this App to view status & results of submitted workflows',
-            image: '/src/img/analysis-icons/gwasResults.svg',
-          };
-          break;
-        default:
-          break;
+              image: '/src/img/analysis-icons/virulence.png',
+            };
+            break;
+          case 'vaGWAS':
```

Comment:
can we just remove this one?

Reply:
vaGWAS?

Reply:
im not sure if its being used anywhere else. probably not? I think removing it should be part of a separate ticket/pr though

Reply:
Ok, good point. Looks like it was added here https://github.com/uc-cdis/data-portal/pull/501/files. I'll add a ticket to the board.

Reply:
https://ctds-planx.atlassian.net/browse/VADC-228